frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -67,9 +72,23 @@ export const CommandPalette: FC = () => {
                 key={table.name}
                 value={table.name}
                 onSelect={() => goToERD(table.name)}
+                data-focused={focusedTableName === table.name}
               >
-                <Table2 className={styles.itemIcon} />
-                {table.name}
+                {/** biome-ignore lint/a11y/useKeyWithClickEvents: Keyboard interaction is implemented in the parent Command.Item component's onSelect handler. */}
+                <div
+                  className={styles.itemInner}
+                  onClick={(event) => {
+                    event.stopPropagation()
+                    setTableName(table.name)
+                    setFocusedTableName((prev) =>
+                      prev === table.name ? null : table.name,
+                    )
+                  }}
+                  onDoubleClick={() => goToERD(table.name)}
+                >
+                  <Table2 className={styles.itemIcon} />
+                  {table.name}
+                </div>
```

Comment:
To overwrite the behaviour of `onClick`, I added this inner element and set `event.stopPropagation()` to avoid calling `onSelect` of `Command.Item` by clicking option.

Reply:
[The "lint/a11y/useKeyWithClickEvents" biome rule](https://biomejs.dev/ja/linter/rules/use-key-with-click-events/) requires some event callbacks for keyboard users.
In this case, `onSelect` of `CommandI.Item` support the behaviour, so that I just ignored the warning.

Reply:
It would be very nice if we could implement `onClick` directly on the `Command.Item` component. However, it looks impossible to overwrite the behaviour because of [the current implementation of `Command.Item` component](https://github.com/pacocoursey/cmdk/blob/fb4ea04e9ec211777fbb39c6104e3c5f2ee107d2/cmdk/src/index.tsx#L718)